tests/plugins/richcombo/richcombo.js

```diff
@@ -56,5 +56,56 @@ bender.test( {
 
 		assert.areEqual( 0, combo._.listeners.length, 'Listeners array is empty.' );
 		assert.isTrue( listenersRemoved, 'All listeners are removed.' );
-	}
+	},
+
+	// (#1268)
+	'test updatestate': testUpdateState(),
+
+	// (#1268)
+	'test updatestate read only editor': testUpdateState( { readOnly: true } ),
+
+	// (#1268)
+	'test updatestate source mode': testUpdateState( { mode: 'source' } ),
```

Comment:
This test does not check if everything works fine in `source` mode. In reality it checks how richcombo behaves in readonly editor. To really check how it behaves in different mode, [`editor#setMode method](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#method-setMode) should be used.